<launch>
	<node pkg="final_challenge" type="PurePursuit.py" name="PurePursuit" output="screen"/>
</launch>
